__test__/validator-base/validate-max.ts

```diff
@@ -0,0 +1,88 @@
+import { describe, it, expect, beforeEach } from "vitest";
```

Comment:
'describe' is declared but its value is never read.

remove describe since its unused